health/micro-ui/web/workbench/Dockerfile

```
@@ -5,20 +1,39 @@
+WORKDIR /app/web
+
+# Show heap size limit
+RUN node -e "console.log('Heap limit (MB):', v8.getHeapStatistics().heap_size_limit / (1024 * 1024))"
+
+# Print a marker log (optional debug message)
+RUN node -e "console.log('core only')"
+
+# Move into microplan and run install script
+WORKDIR /app/web/workbench
+RUN chmod +x ./install-deps.sh
+RUN ./install-deps.sh
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)